src/javascript/JContent/ContentTree/ContentTree.jsx

```diff
@@ -171,6 +172,14 @@ export const ContentTree = ({setPathAction, openPathAction, closePathAction, ite
                       onOpenItem={object => dispatch(openPathAction(object.id))}
                       onCloseItem={object => dispatch(closePathAction(object.id))}
             />
+            {item.treeConfig.showContextMenuOnRootPath &&
+            <div
+                style={{
+                   minHeight: '100%'
```

Comment:
ContentTree has no fixed height, this minHeight is quite random and can give unexpected results

Reply:
Should I put a specific size here?

Reply:
minHeight: '200px',
height: '100%'
The div is in the accordion, it's a sibling of contentTree